### ISSUE TITLE
java.lang.ArrayIndexOutOfBoundsException: 12 

### ISSUE THREAD

**Issue by lays24mc** [Aang23/UndergroundBiomesConstructs#38]
Hi,

when i used a Betterbuilderwands wand with Underground Biomes blocks, my game crashed with this error. 

here my pastebin: https://pastebin.com/Yg5irfGA

Minecraft 1.12.2
Forge 2784

**Comment by Aang23** [comment]
Thanks. I'll be taking a look at this.

**Comment by lays24mc** [comment]
@Aang23, a friend tell me this:

> People using the Builders Wand don't have the Underground Biomes Block in the Hand. So it can't be guaranteed, that the meta matches any Underground Biomes Block meta. You have to check the targeted block, when the Item in the Main Hand isn't a Block from Underground Biomes.

I hope it can help you

**Comment by Aang23** [comment]
Yes that will be helpful. I did have time to look into the issue lately. I'll be working on that in the next few days.
